src/_11ty/filters.js

```diff
@@ -0,0 +1,182 @@
+import { getPageInfo } from './utils/get-page-info.js';
+import { fromHtml } from 'hast-util-from-html';
+import { selectAll } from 'hast-util-select';
+import { toText } from 'hast-util-to-text';
+import { escapeHtml } from 'markdown-it/lib/common/utils.mjs';
+
+export function regexReplace(input, regex, replacement = '') {
+  return input.toString().replace(new RegExp(regex), replacement);
+}
+
+export function toISOString(input) {
```

Comment:
Could we get function comments to explain when/where these filters are used?

Reply:
I'd love to. I'd like to do this as follow-up though as I plan to standardize old and new filters.